src/components/pages/Order/Main/AdminPanel/AdminPanel.js

```diff
@@ -0,0 +1,93 @@
+import React, { useContext, useState } from "react"
+import styled from "styled-components"
+import AdminContext from "../../../../../context/AdminContext"
+import PanelTabButton from "./PanelTabButton"
+import { FaPen, FaPlus, FaChevronDown, FaChevronUp } from "react-icons/fa"
+import { theme } from "../../../../../theme"
+import PanelOption from "./Panel"
+
+const { colors, fonts, spacing } = theme
+
+function AdminPanel() {
+   const { isActiveTab, setIsActiveTab, isPanelVisible, setIsPanelVisible } =
```

Comment:
Alors, avec le "isNanana" ou "hasNanan" c'est en réalité une question qu'on pose en anglais.
```suggestion
   const { isTabActive, setIsTabActive, isPanelVisible, setIsPanelVisible } =
```

Et là, de la même manière que tu as très bien dit "isPanelVisible", ici, on va préférer "isTabActive"